multi/controllers.go

```diff
@@ -537,15 +538,28 @@ func (p *Proxy) PRCProxySingleController(ctx *gin.Context) {
 		ctx.JSON(cmonapi.RequestStatusToStatusCode(resp.RequestStatus), resp)
 		resp.RequestStatus = cmonapi.RequestStatusUnknownError
 		resp.ErrorString = "Single controller is not defined"
+		ctx.JSON(cmonapi.RequestStatusToStatusCode(resp.RequestStatus), resp)
 		return
 	}
 	controller := p.cfg.ControllerById(p.cfg.SingleController)
 	targetURL, _ := url.Parse("https://" + controller.Url + "/v2" + ctx.Param("any"))
-	req, err := http.NewRequest(ctx.Request.Method, targetURL.String(), ctx.Request.Body)
+
+	bodyBytes, err := io.ReadAll(ctx.Request.Body)
```

Comment:
Solution was suggested by AI, i'm not entirely sure why it wasn't working before, but AI suggested:
"The issue likely arises from the way the request body is being handled when passing through the proxy method. In Go, ctx.Request.Body is a io.ReadCloser that can only be read once. Once it is read, it becomes empty for subsequent reads."